content/swiftui/concepts/viewmodifier/terms/tracking/tracking.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: '.tracking()'
+Description: 'Modifies the spacing of the characters in a Text view.'
+Subjects:
+  - 'Mobile Development'
+  - 'iOS'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Modifiers'
+  - 'Xcode'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.tracking()`** modifier method modifies the spacing of the characters in a [`Text`](https://www.codecademy.com/resources/docs/swiftui/views/text) view.
+
+## Syntax
+
+```pseudo
+Text("Some text")
+    .tracking(x)
+```
+
+The `.tracking()` modifier method takes one parameter, `x`, entered as an integer to define the space to be added to or deducted from each character's default spacing in `pt` (points). SwiftUI applies the system's default value if `x` is given `0` as a value or no parameter is given. A positive value for `x` results in additional spacing, whereas a negative value results in decreased spacing.
```

Comment:
Can we break this down into an unordered list?

Reply:
I updated it to a bullet list. 👍🏻